### ISSUE TITLE
Implement "dsl_ode_action_capture_image_player_add/remove" services

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#465]
Want to further simplify the process of playing a captured image by allowing a Player to be created and added to a Capture Action directly.

This should support multiple types of players... Render and RTSP to start with.

* `dsl_ode_action_capture_image_player_add`
* `dsl_ode_action_capture_image_player_remove`

this is dependent on #464

**Comment by rjhowell44** [comment]
Propose API
```c
/**
 * @brief Adds an Image Player, Render or RTSP type, to a named Capture Action.
 * Once added, each captured image's file_path will be added (or queued) with
 * the Image Player to be played according to the Player's settings. The Action can
 * have at most one Image Player.
 * @param[in] name unique name of the Capture Action to update
 * @param[in] player unique name of the Image Player to add
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_ODE_ACTION_RESULT otherwise.
 */
DslReturnType dsl_ode_action_capture_image_player_add(const wchar_t* name, 
    const wchar_t* player);
    
/**
 * @brief Removes an Image Player, Render or RTSP type, from a named Capture Action.
 * @param[in] name unique name of the Capture Action to update
 * @param[in] player unique name of the Image Player to remove
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_ODE_ACTION_RESULT otherwise.
 */
DslReturnType dsl_ode_action_capture_image_player_remove(const wchar_t* name, 
    const wchar_t* player);
```

**Comment by rjhowell44** [comment]
Implementation has been merged into the v0.14.alpha branch, and the `ode_occurrence_object_capture_overlay_image.py` example has been updated to use the new service.
